chalicelib/blueprints/announcements.py

```diff
@@ -0,0 +1,55 @@
+from chalice import Blueprint
+from chalicelib.services.ses import SesMailSender, SesDestination
+from chalicelib.services.mongo import MongoService
+import boto3
+
+announcements_routes = Blueprint(__name__)
+
+ses_client = boto3.client('ses')
+
+
+# why is GET needed?
+@announcements_routes.route("/announcement", methods=["GET", "POST"], cors=True)
+def send_announcement():
+    MongoServer = MongoService()
+    MongoServer.connect()
+    users_data = MongoServer.get_all_data()
+
+    emails = [user['email'] for user in users_data]
+
+    # PLEASE REPLACE WITH YOUR OWN EMAIL FOR TESTING
+    # emails = ['cwilliam.gough@gmail.com']
+
+    request = announcements_routes.current_request
+    request_body = request.json_body
+
+    ses = SesMailSender(ses_client)
+    for email in emails:
+        ses_destination = SesDestination(
+            tos=[email]
+        )
+        ses.send_email(
+            source="techteampct@gmail.com",
+            destination=ses_destination,
+            subject=request_body['subject'],
+            text=request_body['content'],
+            html=request_body['content']
+        )
+
+    return f'Sent emails to {len(emails)} people: {emails}'
+
+
+# @announcements_routes.route("/test-email")
```

Comment:
and if the above works, feel free to get rid of this!

Reply:
I just changed it so you can test just the backend (otherwise the request is empty and no data is sent) so I'll leave it